contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -89,6 +89,33 @@ CREATE OR REPLACE FUNCTION sys.radians(IN arg1 TINYINT)
 RETURNS int  AS 'babelfishpg_tsql','smallint_radians' LANGUAGE C STRICT IMMUTABLE PARALLEL SAFE;
 GRANT EXECUTE ON FUNCTION sys.radians(TINYINT) TO PUBLIC;
 
+CREATE OR REPLACE VIEW information_schema_tsql.tables AS
```

Comment:
as this will show up via minor upgrade, please add a test in upgrade framework and add to 14-6 schedule file